generator/generator.py

```diff
@@ -216,7 +214,7 @@ def main() -> None:
             else:
                 node, solution, kind = res
                 # Compose and print the puzzle
-                puzzle : Dict[str, Any] = {
+                puzzle = {
```

Comment:
Just curious, why this? @niklasf

Reply:
The type hint was required when building the dict in multiple steps, but now that the dict is built from a single literal the type is implied (and a more precise one).